app/views/layouts/application.html.erb

```diff
@@ -1,15 +1,42 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>TemplateRailsStylelineRubocop</title>
+    <title>Events paradise | Join us Today!</title>
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
-
     <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
+    <%= favicon_link_tag asset_path('favicon.png') %>
   </head>
-
   <body>
-    <%= yield %>
+    <% flash.each do |key, value| %>
+      <%= content_tag(:div, class: "alert alert-#{key} alert-dismissable") do %>
+        <%= value %>
+        <button type="button" class="close" data-dismiss="alert" aria-hidden="true">&times;</button>
+      <% end %>
+    <% end %>
+    <nav class="navbar navbar-expand-lg navbar-light bg-primary mb-0">
+      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+      </button>
+      <div class="collapse navbar-collapse" id="navbarSupportedContent">
+        <ul class="navbar-nav mr-auto">
+          <li><%= link_to 'Private Events', root_path, class: 'text-white font-weight-bold' %></li>
+        </ul>
+        <form class="form-inline my-2 my-lg-0">
+          <% if current_user %>
+            <%= link_to "Signed In as " + @user.name, user_path(@user),class: 'text-white ml-3'  %>
+            <%= link_to 'All events', events_path, class: 'btn btn-light my-2 my-sm-0 text-info ml-2' %>
+            <%= link_to 'Create event', new_event_path, class: 'btn btn-light my-2 my-sm-0 text-info ml-3' %>
+            <%= link_to "Sign out", sign_out_path, method: :delete, class: 'btn btn-light my-2 my-sm-0 text-info ml-2' %>
+          <% else %>
+            <%= link_to 'All events', events_path, class: 'btn btn-light my-2 my-sm-0 text-info ml-2' %>
```

Comment:
There are still logic in your code. please, move them into views helpers

Reply:
All the tutorials and reading materials we followed up used if/else statements in their views, also the #any? and #each methods. This is the first time we see that you cannot use IF/ELSE statements in your view. Below is a screenshot of an official RAILCASTA video where they are doing exactly what we are doing in our project.
![20200915_083411](https://user-images.githubusercontent.com/19146884/93174506-be3a2e00-f736-11ea-8400-4a3d436e7f20.jpg)